test/v2/unit/wallet.ts

```diff
@@ -698,4 +698,43 @@ describe('V2 Wallet:', function() {
     }));
   });
 
+  describe('Maximum Spendable', function maximumSpendable() {
+    let bgUrl;
+
+    before(co(function *() {
+      nock.activeMocks().should.be.empty();
+      bgUrl = common.Environments[bitgo.getEnv()].uri;
+    }));
+
+    it('arguments', co(function *() {
+      const optionalParams = {
```

Comment:
Why do we need to pass all these other options?

I think you can just have `recipientAddress` here.

Reply:
We don't have tests for all the other params. If we are gonna test for one, why not the rest?